Libplanet/Net/Swarm.cs

```diff
@@ -1786,7 +1786,7 @@ CancellationToken cancellationToken
                             "Peer [{0}] didn't return any hashes; ignored.",
                             peer.Address.ToHex()
                         );
-                        return workspace;
+                        break;
```

Comment:
What is this change for?

Reply:
It is ok because return `workspace` after the `finally` statement. Other return `workspace` things are all of the return at the finish of this method. So I want to standardize this too.